### ISSUE TITLE
Resisted wrist flexion provokes pain

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10809]
**Preferred term label:**
Resisted wrist flexion provokes pain

**Synonyms**

**Definition (free text, please give PubMed ID)**
Active wrist flexion against resistance increases pain in the arm

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0030836

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Medial epicondylitis
Flexor carpi radialis (FCR) tendonitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001363
- label: Resisted wrist flexion provokes pain
- parents: [Wrist pain (HP:0030836)](https://hpo.jax.org/app/browse/term/HP:0030836)
- pmid: 
- def: Active wrist flexion against resistance increases pain in the arm.
- comment: This sign may be observed with conditions such as medial epicondylitis and lexor carpi radialis tendonitis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10809
- synonyms:
